src/main/java/org/prebid/server/util/HttpUtil.java

```diff
@@ -68,4 +77,27 @@ public static String decodeUrl(String input) {
             throw new IllegalArgumentException(String.format("Cannot decode input: %s", input));
         }
     }
+
+    /**
+     * Creates header from name and value, when value is not null or empty string.
+     */
+    public static void addHeaderIfValueIsNotEmpty(MultiMap headers, String headerName, String headerValue) {
+        if (StringUtils.isNotEmpty(headerValue)) {
+            headers.add(headerName, headerValue);
+        }
+    }
+
+    /**
+     * Crates http headers from {@link Device} properties.
+     */
+    public static MultiMap addDeviceHeaders(MultiMap headers, Device device) {
```

Comment:
m.b. it is too specific logic about device headers to add it to util class. Http utils must aware on HTTP related things only.
i think it would be better to leave this method it in bidder's class, even it duplicates.